scripts/run_olserver.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# This script git pulls master for openlibrary, infogami, and olsystem
+# and then starts the Docker service specified by $SERVICE
+
+cd /opt/olsystem
+sudo git pull origin master
+cd /opt/openlibrary
+sudo git pull origin master
+cd /opt/openlibrary/vendor/infogami
+sudo git pull origin master
+cd /opt/openlibrary
+
+export SERVICE=${SERVICE:-"web"}  # options: web, covers, infobase, home
+echo "Starting $SERVICE"
+cd /opt/openlibrary
+sudo docker-compose build --pull $SERVICE
+sudo docker-compose down
+sudo docker-compose up -d --no-deps memcached
```

Comment:
This will need to be behind a config; we don't always want a `memcached` running on each node.